core/shortcut_registry.js

```diff
@@ -22,16 +23,16 @@ goog.requireType('Blockly.Workspace');
 /**
  * Class for the registry of keyboard shortcuts. This is intended to be a
  * singleton. You should not create a new instance, and only access this class
- * from Blockly.ShortcutRegistry.registry.
+ * from ShortcutRegistry.registry.
  * @constructor
  */
-Blockly.ShortcutRegistry = function() {
+const ShortcutRegistry = function() {
   // Singleton instance should be registered once.
-  Blockly.ShortcutRegistry.registry = this;
+  ShortcutRegistry.registry = this;
```

Comment:
Having the constructor overwrite the singleton seems like a bad idea (especially given the constructor is exported!)  No fix required now, but see below.